typespec-tests/pom.xml

```diff
@@ -42,6 +42,11 @@
       <artifactId>azure-identity</artifactId>
       <version>1.8.2</version>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-experimental</artifactId>
+      <version>1.0.0-beta.39</version>
```

Comment:
this version currently is only local build, from https://github.com/Azure/azure-sdk-for-java/pull/34525